lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java

```diff
@@ -3569,7 +3586,25 @@ protected static boolean makeLink(UIContainer container, String ID, SimplePageIt
 		return available;
 	}
 
-	private static void disableLink(UIComponent link, MessageLocator messageLocator) {
+	private static String getUserDisplayName(String owner) {
+
+		if (owner== null) {
+			return "";
+		}
+
+		User user = null;
+		try {
+			user = UserDirectoryService.getUser(owner);
+		} catch (UserNotDefinedException e) {
+			log.warn("Owner #: " + owner + " does not have an associated user.");
```

Comment:
Shouldn't be a warn as user's getting deleted is expected and the content will still be owned.